redux/features/prodInfo-slice.js

```diff
@@ -4,8 +4,14 @@ const initialState = {
   visibleImage:
     "https://m.media-amazon.com/images/I/A13usaonutL._CLa%7C2140%2C2000%7C81t54H7ZpXL.png%7C0%2C0%2C2140%2C2000%2B0.0%2C0.0%2C2140.0%2C2000.0_AC_UX385_.png",
   pageUrl: "http://localhost:3000/#",
-  currentImage:
+  currentImage: {
+    url: "https://m.media-amazon.com/images/I/A13usaonutL._CLa%7C2140%2C2000%7C81t54H7ZpXL.png%7C0%2C0%2C2140%2C2000%2B0.0%2C0.0%2C2140.0%2C2000.0_AC_UX385_.png",
+    isProd: true,
+  },
+  imageBtnArr: [
     "https://m.media-amazon.com/images/I/A13usaonutL._CLa%7C2140%2C2000%7C81t54H7ZpXL.png%7C0%2C0%2C2140%2C2000%2B0.0%2C0.0%2C2140.0%2C2000.0_AC_UX385_.png",
+    "https://m.media-amazon.com/images/I/71deJMlACUL._AC_UX466_.jpg",
+  ],
```

Comment:
changed currentImage state to an obj with the url and weither its the product, or not. Used to determine if when the shirtColor button is hovered, will it display it or not.

added imageBtnArray to redux state as it will be used across your shirtColor component and my imageDisplay component